charts/astronomer/templates/_helpers.yaml

```diff
@@ -139,6 +135,30 @@
 {{- end }}
 {{- end }}
 
+{{ define "houston.bootstrapperServiceAccount" -}}
+{{- if .Values.houston.serviceAccount.create -}}
+{{ default (printf "%s-houston-bootstrapper" .Release.Name ) .Values.houston.serviceAccount.name }}
+{{- else -}}
+    {{ default "default" .Values.houston.serviceAccount.name }}
+{{- end }}
+{{- end }}
```

Comment:
@danielhoherd @Shubham-astro i made this change should we adopt to all other templates as well . please add your views on the comment

Reply:
So, if block is fine, else block means if houston.serviceAccount.name is not defined it defaults to "default" and the template will use the existing "default" service account in the namespace, I think we can use this for other SA's.

Reply:
I definitely think we should be consistent wherever possible.

I also like how this template keeps the logic in the helm template and avoids template logic in the kubernetes manifest.

I say we should adopt this wherever possible.

What may be even better is if we can create a single function that would handle all of these duplicate template definitions.